localstack-core/localstack/services/lambda_/api_utils.py

```diff
@@ -637,7 +637,7 @@ def validate_and_set_batch_size(service: str, batch_size: Optional[int] = None)
     BATCH_SIZE_RANGES = {
         "kafka": (100, 10_000),
         "kinesis": (100, 10_000),
-        "dynamodb": (100, 1_000),
+        "dynamodb": (100, 10_000),
```

Comment:
https://docs.aws.amazon.com/lambda/latest/api/API_CreateEventSourceMapping.html#API_CreateEventSourceMapping_RequestBody

> **Amazon DynamoDB Streams** – Default 100. Max 10,000.

Reply:
@gregfurman Can we trigger the ext pipeline here to check whether it breaks pipes?

(if urgent, better compromise at Pipes in preview rather than ESM)